spec/javascript/components/react_tests_spec.js

```diff
@@ -11,7 +11,14 @@ describe('train component', () => {
 
   beforeEach(() => {
```

Comment:
Please rename this file -- your enzyme tests should be named by the component being tested